resources/lang/en_US/default.po

```diff
@@ -449,7 +449,7 @@ msgid "settings.certificates.ifsg_confirmed.hint"
 msgstr "Your health instruction has been confirmed, so you can no longer change your data."
 
 msgid "settings.certificates.confirmed.info"
-msgstr "You personally checked the certificate / license"
+msgstr "You personally checked the certificate / license and it mets the requirements"
```

Comment:
Do you mean "meets the requirements"?

Reply:
I mean "coding while half asleep might sometimes lead to typing mistakes" :upside_down_face: